Guides/TestingGuide.md

```diff
@@ -0,0 +1,41 @@
+## Testing
+- setup environment
+```
+    python3 -m venv env
+    source env/bin/activate
+```
+- install requirements
+```
+    pip install -r req.txt
+```
+- set SecretKey
+```
+    export STRIPE_API_KEY=12345
+```
+- create Database
+```
+    python
+    from app import db
+    db.create_all()
+```
+- start app
+```
+  python app.py
+```
+
```

Comment:
Add these info in the readme why to add it in the seperate file

Reply:
@Geek-ubaid I have added these to readme but with application run perspective,
here I have included many things like what behaviour to expect and much more, please look at the complete file for complete understanding as it's not showing complete here(for me atleast).